code/workspaces/client-api/src/schema/resolvers.js

```diff
@@ -45,13 +44,18 @@ const resolvers = {
     createStack: (obj, { stack }, { user, token }) => permissionChecker(STACKS_CREATE, user, () => stackApi.createStack({ user, token }, DATALAB_NAME, { projectKey: PROJECT_KEY, ...stack })),
     deleteStack: (obj, { stack }, { user, token }) => permissionChecker(STACKS_DELETE, user, () => stackApi.deleteStack({ user, token }, DATALAB_NAME, { projectKey: PROJECT_KEY, ...stack })),
     createDataStore: (obj, { dataStore }, { user, token }) => (
-      permissionChecker(STORAGE_CREATE, user, () => dataStoreApi.createDataStore({ user, token }, DATALAB_NAME, { projectKey: PROJECT_KEY, ...dataStore }))
+      // permissionChecker(STORAGE_CREATE, user, () => dataStoreApi.createDataStore({ user, token }, DATALAB_NAME, { projectKey: PROJECT_KEY, ...dataStore }))
```

Comment:
Just checking whether this has been left in accidentally

Reply:
It has indeed been left in accidentally, I'll remove